test/eventlistener_test.go

```diff
@@ -245,8 +266,10 @@ func TestEventListenerCreate(t *testing.T) {
 		Spec: v1alpha1.PipelineResourceSpec{
 			Type: "git",
 			Params: []v1alpha1.ResourceParam{
-				{Name: "body", Value: strings.Replace(`{"one": "zonevalue", "two": {"name": "zfoo", "value": "bar"}}`, " ", "", -1)},
```

Comment:
I think this `zonevalue` came from the previous body, so we'd have to replace all instances of it